utils/sampler/lib/constants.dart

```diff
@@ -0,0 +1,8 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Route names.
+const String kNewSampleSelectView = '/newSampleSelectView';
+const String kDetailView = '/detailView';
+const String kNewSampleView = '/newSampleView';
```

Comment:
Minor style thing: perhaps add 'route' to these names (e.g. kRouteNewSampleSelectView) so that at a glance you know it is route name and not a user facing label or something.

Reply:
Done.